tests/baseTestValidator.py

```diff
@@ -86,7 +86,7 @@ def tearDownClass(cls):
         cls.interfaces.close()
         dropDatabase(cls.interfaces.jobDb.dbName)
         dropDatabase(cls.interfaces.errorDb.dbName)
-        dropDatabase(cls.interfaces.stagingDb.dbName)
```

Comment:
@jworcestBAH Is there a reason this is commented out whereas it was uncommented previously?